Client/src/Views/UserDatasets/UserDatasetTutorial.jsx

```diff
@@ -1,32 +1,48 @@
 import React from 'react';
 import { bytesToHuman } from 'wdk-client/Utils/Converters';
+import { Link } from 'wdk-client/Components';
 
 import tutStep2 from './images/tut-step-2.jpg';
 import tutStep3 from './images/tut-step-3.jpg';
 
 function UserDatasetTutorial ({ projectName, quotaSize }) {
   const galaxyUrl = 'https://eupathdb.globusgenomics.org/';
-  return projectName === 'MicrobiomeDB' ? <div/> : (
-    <div className="row UserDataset-Tutorial">
-      <div className="box xs-12 md-6">
-        <h2>EuPathDB Galaxy</h2>
-        <img src={tutStep2} />
+  return projectName === 'MicrobiomeDB' ? (
+    <div className="UserDataset-TutorialMicrobiome">
+      <div className="UserDataset-TutorialMicrobiome__lead">You do not have any data sets.</div>
+      <div>You can add your data sets to {projectName} using the <Link to="/workspace/datasets/new-upload">upload tool</Link>.</div>
```

Comment:
This link is incorrect and doesn't work, should be `datasets/new`.

"The upload tool" might be confusing for someone who sees this for the first time, since they neccesarily won't know which tool is referred.

Consider more cheerful verbiage?
`Try adding a data set using the <a href="/workspace/datasets/new-upload">New Upload section </a> above!`